waspc/data/Generator/templates/sdk/wasp/package.json

```diff
@@ -9,6 +9,9 @@
       "types": "tsc --declaration --emitDeclarationOnly --stripInternal --declarationDir dist"
     },
   "exports": {
+      "./client/operations/queryClient": {
+        "client-runtime": "./dist/client/operations/queryClient.js"
+      },
```

Comment:
This how we expose the SDK's private API to the framework, while hiding it from the user.

Reply:
Great stuff! I didn't know that `exports` API worked this way. I always thought that the `import` and `require` were the only options.